Resources/public/js/views/services/plugins/ez-searchplugin.js

```diff
@@ -324,21 +325,26 @@ YUI.add('ez-searchplugin', function (Y) {
         _doLocationSearch: function (e) {
             var search = Y.merge(e.search),
                 listView = e.target,
-                attrs = {'loadingError': false};
+                attrs = {'loadingError': false},
+                callback = function (error, result, resultCount) {
+                    attrs.loadingError = error ? true : false;
```

Comment:
hum it's more a matter of taste than anything but the ternary version is maybe a tiny bit more readable :)